README.md

```diff
@@ -119,6 +119,29 @@ The project is set up with [ESLint](https://eslint.org/) for JavaScript/TypeScri
 
 To ease development, we highly recommend enabling ESLint within your editor so you can see issues in real time. If you're using Visual Studio Code, you can use the [VS Code ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint). For other editors, see [available ESLint integrations](https://eslint.org/docs/user-guide/integrations).
 
+## Internationalization (i18n)
+
+Simulator leverages gettext PO files to create a `i18n.json` file located in `/i18n/i18n.json`. The source files are scanned for imports of `@i18n`, and uses of the default exported function (`tr` by convention) are detected and inserted into PO files located in `/i18n/po/`. These PO files are "built" into the JSON file, which is then injected into the frontend via webpack's `DefinePlugin` (see `configs/webpack/common.js` for details). The `tr` function reads this object at runtime to make translations available.
+
+To update the PO files, run `yarn run generate-i18n`. While the generation script *should* preserve your work-in-progress, it is recommended to commit, stash, or otherwise backup the PO files prior to running this script.
```

Comment:
What's the expectation for devs regarding PO files? Should `generate-i18n` be run before every PR?

For example, if I add a new UI string, should I regenerate the PO files and include the modified PO files in my PR, even if I don't fill in the translations?

Reply:
`generate-i18n` probably should be run before PRs, but the result is the same; If no translations exist the fallback is always `EN_US`, which is specified directly in the source code.

Reply:
Makes sense. I worry about how quickly our "% of UI translated" will drop considering how rapidly our UI changes. May need to come up with some periodic update with translators